demo-iota/README.md

```diff
@@ -1,23 +1,65 @@
 # Iota
 
-Iota is a toy functions-as-a-service platform where all the functions are represented as Wasm modules.
+Iota is a toy functions-as-a-service platform where all the functions are
+represented as Wasm modules.
 
-Iota maps the HTTP body to the Wasm function's standard input. And it maps the Wasm function's standard output to the HTTP response.
+Iota maps the HTTP body to the Wasm function's standard input. And it maps the
+Wasm function's standard output to the HTTP response.
+
+This document describes the high-level make command interface to use and
+configure an Iota service, as well as the low-level APIs implemented by each
+Iota backend.
+
+## Iota Proxy Service
+
+In order to simplify operation of these Iota services, there is a reverse proxy
+deployed that can reach all Iotas through a single entrypoint.
+
+### Upload a module:
+
+`POST https://iota.dylibso.com/upload/<app>/<name>`
+
+Where `<app>` is any of `go`, `rust`, `node`, `deno`, and the `<name>` is the
```

Comment:
I think it could be a little confusing about which part is the "app". Maybe it make sense to call this "host" or add some language in the doc that it's all iota but just different host languages.